frontend/src/bundles/common/components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+
+import logo from '~/assets/img/logo.svg';
+
+import { AppRoute } from '../../enums/app-route.enum.js';
+import {
+    GoalsIcon,
+    HelpIcon,
+    LogoutIcon,
+    OverviewIcon,
+    ScheduleIcon,
+    WorkoutIcon,
+} from '../icons/sidebar-icons/sidebar-icons.js';
+import { SidebarNav } from '../sidebar-nav/sidebar-nav.js';
+
+const Sidebar: React.FC = () => {
+    const { pathname } = useLocation(); // Get the current pathname
+
+    // Example usage of useState to track active state for each route
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const [activeRoute, setActiveRoute] = useState(pathname);
```

Comment:
Remove the comment here and also was disabling the linter really necessary?

Reply:
Mm I just need the state of the initial load, so I don't need to change it on an effect, so I was getting error for not using the setActiveRoute. 
I will take any suggestion to solve it